### DISCUSSION TITLE
Created on Solsea but did not Mint, however NFT now shows in phantom wallet

### DISCUSSION THREAD

**Discussion by TFSLOAN** [deso-protocol/dips#208] [Q&A]
Curios what has happened to me here,
I created a NFT on solsea when I went to mint, the wallet I used was empty so I canceled vs approve the transaction.
On Solsea the collection and wallet show no NFTs

but on Magic Eden, in the same Phantom wallet, I see the NFT Is in the wallet.

Is this a form of lazy minting on Solana? and how do I access the NFT again on Solsea to mint it?

TY for the help.

**Comment by dgsus** [comment]
i think you are lost sir. not sure how deso is in anyway related to solana...

**Reply by TFSLOAN** [reply]
ty 
I realize I ended up here from another solsea thread.
